go/billing/settings.py

```diff
@@ -56,6 +56,11 @@
 LOW_CREDIT_NOTIFICATION_EMAIL = getattr(
     settings, 'BILLING_LOW_CREDIT_NOTIFICATION_EMAIL', 'support@vumi.org')
 
+PROVIDERS = getattr(
+    settings, 'BILLING_PROVIDERS', (
+        ('mtn', 'MTN'),
+        ('vodacom', 'Vodacom')))
```

Comment:
Hmm. Changing that might require a migration so perhaps we should add it in this PR?